aicsimageio/readers/tiff_reader.py

```diff
@@ -42,6 +75,156 @@ def _is_this_type(buffer: io.BufferedIOBase) -> bool:
                     return False
             return True
 
+    @staticmethod
+    def _imread(
+        img: Path,
+        scene: int,
+        page: int
+    ) -> np.ndarray:
+        # Load Tiff
+        with TiffFile(img) as tiff:
+            # Get proper scene
+            scene = tiff.series[scene]
+
+            # Get proper page
+            page = scene.pages[page]
+
+            # Return numpy
+            return page.asarray()
+
+    @property
+    def dask_data(self) -> da.core.Array:
+        """
+        Read a TIFF image file as a delayed dask array where each chunk of the constructed array is a delayed YX plane.
+
+        Returns
+        -------
+        img: dask.array.core.Array
+            The constructed delayed YX plane dask array.
+        """
+        if self._dask_data is None:
+            # Load Tiff
+            with TiffFile(self._file) as tiff:
+                # Check each scene has the same shape
+                # If scene shape checking fails, use the specified scene and update operating shape
+                scenes = tiff.series
+                operating_shape = scenes[0].shape
+                for scene in scenes:
+                    if scene.shape != operating_shape:
+                        operating_shape = scenes[self.specific_s_index].shape
+                        scenes = [scenes[self.specific_s_index]]
+                        log.info(
+                            f"File contains variable dimensions per scene, "
+                            f"selected scene: {self.specific_s_index} for data retrieval."
+                        )
+
+                # Get sample yx plane
+                sample = scenes[0].pages[0].asarray()
+
+                # Combine length of scenes and operating shape
+                # Replace YX dims with empty dimensions
+                operating_shape = (len(scenes), *operating_shape)
+                operating_shape = operating_shape[:-2] + (1, 1)
+
+                # Make ndarray for lazy arrays to fill
+                lazy_arrays = np.ndarray(operating_shape, dtype=object)
+                for all_page_index, np_tuple in enumerate(np.ndenumerate(lazy_arrays)):
+                    # Unpack np_tuple
+                    np_index, _ = np_tuple
```

Comment:
Same here, unless you need `np_tuple` explicitly this can be `for all_page_index, (np_index, _) in ...`